c8/redis/core.py

```diff
@@ -12,12 +12,6 @@ def build_request(collection, data):
     return request
 
 
-def response_handler(request, response):
-    if not response.is_success:
-        raise RedisServerError(response, request)
-    return response.body
-
-
 class RedisServerError(C8ServerError):
```

Comment:
A question here should the RedisServerError be present in redis/core file or should it be present in exceptions.py file?

Reply:
That `exceptions.py` is full of one-liners that inherit the same classes (C8ClientError or C8ServerError). I would like to separate and refactor that file. This is a start Redis package has Redis Interface, core, and utils. The same would be excellent for queries, streams, workers, collections etc.